README.md

```diff
@@ -2,6 +2,21 @@
 
 `@metamask/controllers` is a collection of platform-agnostic modules for creating secure data models for cryptocurrency wallets.
 
+## Release & Publishing
```

Comment:
Was this intentionally placed before the table of contents? It would seem more appropriate to place it down where it used to be.